packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -280,13 +280,38 @@ export async function save(ctx: UserCtx<Config>) {
   }
 }
 
+const parseScopedConfigFileUrls = (cfg: any, type: string) => {
```

Comment:
I think it's safe to make this an OIDC by default function, the impl is quite specific. Could go with `enrichOIDCLogos` and update the consumers, also giving us some type safety (we should try to stay from `any` as much as possible)

```ts
function enrichOIDCLogos(config: OIDCLogosConfig) {
   ...
}
```

```ts
function find(ctx: UserCtx) {
   ...
   if (type === ConfigType.OIDC_LOGOS) {
      enrichOIDCLogos(config) // shouldn't need to re-assign here
   }
   ...
}
```

```ts
function publicOidc(ctx: Ctx<void, GetPublicOIDCConfigResponse>) {
   if (scopedConfig) {
      enrichOIDCLogos(scopedConfig)
   }
}
```


I think this is somewhat easier to read. Would also be great if we could add a helper for retrieving the logos doc in `publicOidc`, instead of using `configs.getConfig`. Could add this to here: https://github.com/Budibase/budibase/blob/fe9a91ae7ae2a8882ffa768370935fdd4f2014ee/packages/backend-core/src/configs/configs.ts#L192 meaning we can return a typed config rather than a generic one.